common-crafting.lic

```diff
@@ -88,6 +88,16 @@ module DRCC
     end
   end
 
+  def find_enchanting_room(hometown, override = nil)
+    if override
+      DRCT.walk_to(override)
+    else
+      enchantingrooms = get_data('crafting')['artificing'][hometown]['brazier-rooms']
+      idle_room = get_data('crafting')['artificing'][hometown]['idle-room']
+      DRCT.find_sorted_empty_room(shapingrooms, idle_room)
```

Comment:
❓ Should `shapingrooms` be `enchantingrooms `?

Reply:
Fixed. :)